index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Your Readme Looks Professional, but kindly update the Getting Started Section with the details on how to set up your project locally, you can remove the things that are not required for this project like Run tests, etc.